Dockerfile.controller

```diff
@@ -0,0 +1,23 @@
+# Using the `rust-musl-builder` as base image, instead of 
+# the official Rust toolchain
+FROM  clux/muslrust:1.66.0-stable AS chef
+USER root
+RUN cargo install cargo-chef
+WORKDIR /app
+
+FROM chef AS planner
+COPY . .
+RUN cargo chef prepare --recipe-path recipe.json
```

Comment:
Are we using recipe.json correctly? or should there be a separate for the controller and service.